src/handwritten/_lib.py

```diff
@@ -148,39 +151,72 @@ def task_handle(self) -> type:
     @property
     def cal_handle(self) -> type:
         return CalHandle
-
+    
+    @property
+    def encoding(self):
+        if self._encoding is None:
+            self._import_lib()
+        return self._encoding
+        
     def _import_lib(self):
         """
         Determines the location of and loads the NI-DAQmx CAI DLL.
         """
         self._windll = None
         self._cdll = None
+        self._encoding = None
 
         windll = None
         cdll = None
-
-        if sys.platform.startswith('win') or sys.platform.startswith('cli'):
-            try:
-                if 'iron' in platform.python_implementation().lower():
-                    windll = ctypes.windll.nicaiu
-                    cdll = ctypes.cdll.nicaiu
-                else:
-                    windll = ctypes.windll.LoadLibrary('nicaiu')
-                    cdll = ctypes.cdll.LoadLibrary('nicaiu')
-            except (OSError, WindowsError) as e:
-                raise DaqNotFoundError(_DAQ_NOT_FOUND_MESSAGE) from e
+        encoding = None
+
+        # Feature Toggle to load nicaiu.dll or nicai_utf8.dll
+        # The Feature Toggle can be set in the .env file
+        nidaqmx_c_library = config('NIDAQMX_C_LIBRARY', cast=str, default=None)
+
+        if sys.platform.startswith('win') or sys.platform.startswith('cli'):   
+            if nidaqmx_c_library is not None:
```

Comment:
If we're not going to look at the env var on Linux, then let's only query it on Windows.
```suggestion
        if sys.platform.startswith('win') or sys.platform.startswith('cli'):   
            # Feature Toggle to load nicaiu.dll or nicai_utf8.dll
            # The Feature Toggle can be set in the .env file
            nidaqmx_c_library = config('NIDAQMX_C_LIBRARY', cast=str, default=None)

            if nidaqmx_c_library is not None:
```